Compiler/src/ProgramBuilder.ml

```diff
@@ -59,6 +59,9 @@ type unary_op = Not
     | PreDec
     | PostDec
 
+let unknown_type_int = Int32.sub (Int32.shift_left 1l 12) 1l
```

Comment:
Please add a short comment that these numbers must be kept in sync with TypeSystem.swift.
Also it would probably be nice to have similar Type constructors as [those provided in TypeSystem.swift](https://github.com/googleprojectzero/fuzzilli/blob/master/Sources/Fuzzilli/FuzzIL/TypeSystem.swift#L112). On the other hand, it looks like you only have three places that create types currently in the compiler, so how about just adding a TODO comment here about creating proper type constructors eventually?